README.md

```diff
@@ -64,6 +68,6 @@ Licensed under the Apache License, Version 2.0: http://www.apache.org/licenses/L
 
 ## Security
 
-Please report sensitive security issues via Spotify's [bug-bounty program](https://hackerone.com/spotify) rather than GitHub.
+Please report sensitive security issues using Spotify's [bug-bounty program](https://hackerone.com/spotify) rather than GitHub.
```

Comment:
```suggestion
Please report sensitive security issues through Spotify's [bug-bounty program](https://hackerone.com/spotify) rather than GitHub.
```

i wasn't thinking `via` was half bad actually ... but OK, how about `through`

Reply:
I think using is better